src/hyperion/device_setup_plans/setup_zebra.py

```diff
@@ -20,6 +23,24 @@
 from hyperion.log import LOGGER
 
 
+def bluesky_retry():
```

Comment:
nit: Might be useful to have a comment saying that this is a decorator used to retry a bluesky plan upon exception